src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -7,10 +7,8 @@
 
   <!-- Prevent Razor files from being packaged as bundle resources in iOS/MacCatalyst targets by declaring them
        as RazorComponent instead of Content. -->
-  <ItemGroup>
+  <ItemGroup Condition="'$(DesignTimeBuild)' != 'true'">
```

Comment:
I wouldn't super bother, but if you'd like you could consider using adding these conditions to these items: https://github.com/dotnet/sdk/blob/main/src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.props#L82